cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -589,12 +594,12 @@ type serviceAccountTokenControllerStarter struct {
 
 func (c serviceAccountTokenControllerStarter) startServiceAccountTokenController(ctx ControllerContext) (http.Handler, bool, error) {
 	if !ctx.IsControllerEnabled(saTokenControllerName) {
-		klog.Warningf("%q is disabled", saTokenControllerName)
+		klog.InfoS("Controller is disabled", "name", saTokenControllerName)
```

Comment:
Suggest: `klog.InfoS("saTokenController is disabled", "saTokenControllerName", saTokenControllerName)`